client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx

```diff
@@ -47,23 +46,25 @@ class Locked extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
-		const { privateDomain } = getSelectedDomain( this.props );
+		const { translate, selectedSite } = this.props;
+		const { domain: domainName, privateDomain, registrar } = getSelectedDomain( this.props );
 
 		return (
 			<div>
 				<Card className="transfer-out__card">
 					<p>
-						{ privateDomain
+						{ privateDomain && registrar === registrarNames.WWD
 							? translate(
-									'To transfer your domain, we must unlock it and remove Privacy Protection. ' +
-										'Your contact information will be publicly available during the transfer period.'
+									'To transfer your domain, we must unlock it and remove Privacy Protection. Your contact information will be publicly available during the transfer period. The domain will remain unlocked and your contact information will be publicly available until the transfer is canceled or completed.'
 							  )
-							: translate( 'To transfer your domain, we must unlock it.' ) }{ ' ' }
+							: translate(
+									'To transfer your domain, we must unlock it. It will remain unlocked until the transfer is canceled or completed.'
+							  ) }{ ' ' }
 						<a href={ TRANSFER_DOMAIN_REGISTRATION } target="_blank" rel="noopener noreferrer">
 							{ translate( 'Learn More.' ) }
```

Comment:
Not related to your changes but I'm tempted to think that `Learn More.` should be written `Learn more.` given it used at the end of a paragraph.

Reply:
Makes sense 👍